modules/swagger-codegen/src/main/resources/JavaSpring/api.mustache

```diff
@@ -71,7 +71,7 @@ public interface {{classname}} {
 
 {{/useSpringCloudClient}}
 {{^useSpringCloudClient}}
-    {{#jdk8}}default {{/jdk8}}{{#responseWrapper}}{{.}}<{{/responseWrapper}}ResponseEntity<{{>returnTypes}}>{{#responseWrapper}}>{{/responseWrapper}} {{operationId}}({{#allParams}}{{>queryParams}}{{>pathParams}}{{>headerParams}}{{>bodyParams}}{{>formParams}},{{/allParams}} @RequestHeader(value = "Accept", required = false) String accept) throws Exception{{^jdk8}};{{/jdk8}}{{#jdk8}} {
+    {{#jdk8}}default {{/jdk8}}{{#responseWrapper}}{{.}}<{{/responseWrapper}}ResponseEntity<{{>returnTypes}}>{{#responseWrapper}}>{{/responseWrapper}} {{operationId}}({{#allParams}}{{>queryParams}}{{>pathParams}}{{>headerParams}}{{>bodyParams}}{{>formParams}}{{#hasMore}},{{/hasMore}}{{/allParams}}{{^interfaceOnly}}, @RequestHeader(value = "Accept", required = false) String accept{{/interfaceOnly}}) throws Exception{{^jdk8}};{{/jdk8}}{{#jdk8}} {
```

Comment:
@wingsofovnia  please, update sample Spring projects (run bin/spring-all-pestore.sh) and commit the changes so CI can verify them.

Reply:
@lukoyanov This change doesn't touch any petstore sample. None of it is `interfaceOnly`.